tgg/research/gnomad_tgps.py

```diff
@@ -52,111 +62,125 @@
 
 
 def get_pangenome_ids(
-        id_path: str,
-) -> hl.expr.expressions.typed_expressions.SetExpression:
+    id_path: str,
+) -> hl.expr.SetExpression:
     """
-    Generate a Hail set of sample IDs in the pangenome.
+    Import Hail set of pangenome sample IDs from specified file path.
 
-    :param str id_path: Cloud path to read in the .txt file containing pangenome sample
-    IDs.
-    :return: A set of sample IDs as hl.expr.expressions.typed_expressions.SetExpression.
+    :param str id_path: Path to text file (stored in Google bucket) containing
```

Comment:
```suggestion
    :param id_path: Path to text file (stored in Google bucket) containing
```
just removing the type here since it's redundant